libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialogBot.cs

```diff
@@ -125,6 +126,18 @@ private void SetUpTurnState(ITurnContext turnContext, BotFrameworkClient botFram
             turnContext.TurnState.Add(_languagePolicy);
             turnContext.TurnState.Add(_telemetryClient);
 
+            // Catch "SetTestOptions" event and save into "Conversation.TestOptions".
+            if (turnContext.Activity.Type == ActivityTypes.Event)
+            {
+                var eventActivity = turnContext.Activity.AsEventActivity();
+                if (eventActivity.Name == "SetTestOptions")
+                {
+                    var conversationState = turnContext.TurnState.Get<ConversationState>();
+                    var property = conversationState.CreateProperty<object>("TestOptions");
```

Comment:
@Stevenic, you said we use a different naming convention for "properties" what would be the right name here?